rust/sbp/src/wire_format.rs

```diff
@@ -178,7 +180,10 @@ impl WireFormat for f64 {
 }
 
 #[derive(Debug, Clone)]
-pub struct PayloadParseError {}
+pub struct PayloadParseError {
+    /// A vec that just contains all the bytes
+    pub raw_invalid_payload_bytes: Vec<u8>,
```

Comment:
Variable name is a mouthful, would this work?

```suggestion
    pub raw_bytes: Vec<u8>,
```

Reply:
done